docs/validators/staking/staking-smart-contract.md

```diff
@@ -36,9 +36,9 @@ Because this transaction is a call to the Staking smart contract, it passes info
 - `stake` is the name of the smart contract function to be called;
 - `0002` is the number of nodes (unsigned integer, hex-encoded);
 - 45e7131ba....294812f004 is the BLS key of the first node, represented as a 192-character-long hexadecimal string;
```

Comment:
When I wrote this line, I forgot to add backticks here for formatting, please apply this change:
```suggestion
- `45e7131ba....294812f004` is the BLS key of the first node, represented as a 192-character-long hexadecimal string;
```